okta/resource_okta_user_base_schema_test.go

```diff
@@ -13,11 +14,21 @@ import (
 const baseTestProp = "firstName"
 
 func sweepUserBaseSchema(client *testClient) error {
-	_, _, err := client.apiSupplement.GetUserSchema()
+	var errorList []error
```

Comment:
@bogdanprodan-okta This just to note, Okta SDK should handle this and not the extend SDK.